SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialNetDriver.h

```diff
@@ -85,6 +90,7 @@ class SPATIALGDK_API USpatialNetDriver : public UIpNetDriver
 
 public:
 	USpatialNetDriver(const FObjectInitializer& ObjectInitializer);
+	USpatialNetDriver(FVTableHelper& Helper);
```

Comment:
Hmm Unreal has a big `Don't use` around this functionality -
```
#define DECLARE_VTABLE_PTR_HELPER_CTOR(API, TClass) \
	/** DO NOT USE. This constructor is for internal usage only for hot-reload purposes. */ \
```

Reply:
I'ts more of a "Do not call". I expect this macro to be used in generated code.